tests/poster_mock.py

```diff
@@ -18,28 +18,37 @@ def free_port():
     sock.close()
     return port
 
-def start_server(port, locations=None):
+def start_server(port, **kwargs):
     """Starts Poster mock server
-
     :param port: Port
-    :param locations: Location data
+    :param kwargs:
```

Comment:
@Vatavuk I think we don't need kwargs here, *args, **kwargs are standart signature for python class methods